gradle/warnings.gradle

```diff
@@ -16,6 +16,6 @@
  */
 
 tasks.withType(JavaCompile) {
-  options.compilerArgs << '-Xlint:unchecked' << "-Werror"
-  options.deprecation = true
+//  options.compilerArgs << '-Xlint:unchecked' << "-Werror"
+//  options.deprecation = true
 }
```

Comment:
You probably didn't mean to leave these commented out?

Reply:
ew, thanks for catching that Jens.  I was struggling with the Gradle upgrade in IJ when I made that change